public/js/monitor.js

```diff
@@ -51,68 +67,34 @@ function doOauth(el, {emailWatch = false} = {}) {
     return;
   }
 
+  const lastScannedEmail = sessionStorage.getItem("lastScannedEmail");
+
+  if (typeof emailWatch !== "boolean") {
+    throw new Error("invalid argument option in doOauth");
```

Comment:
And someday we all use typescript just like Greg always said we should.

Reply:
🎯  

cc/ @gregtatum